.github/workflows/aspnetcore-sync-checkdiff.ps1

```diff
@@ -0,0 +1,3 @@
+# Check the code is in sync
+$changed = (select-string "nothing to commit" artifacts\status.txt).count -eq 0
```

Comment:
Is this extra file needed? We could just inline this command in the gh action?

Also, I wonder if using `git diff` would be easier? Something like

```
git diff -M -C -b --ignore-cr-at-eol --ignore-space-at-eol --exit-code > ..\artifacts\diff.txt
$changed = !$?
```

The only issue I see with `git diff` are untracked files, so we would need to run `git add -N .` before running the command. Just an idea to consider.

Reply:
I guess the aspnetcore version of this file did a lot more to open and comment on the github issue. Inlining just this part might be easier now.
https://github.com/dotnet/aspnetcore/blob/main/.github/workflows/ReportDiff.ps1